### DISCUSSION TITLE
Thumbnail Authentication

### DISCUSSION THREAD

**Discussion by stu-ball** [OvenMediaLabs/OvenMediaEngine#1092] [Ideas] [answered]
It appears that the thumbnails are accessible to anonymous users. Is this the intended behavior?

Are there any plans to provide the option to require authentication to access the thumbnails, either using the API authentication or the admission webhook perhaps?

I'm currently using the configuration specified here: https://airensoft.gitbook.io/ovenmediaengine/thumbnail-experiment

**Comment by getroot** [comment]
It's on our backlog, but it's low priority.

**Comment by stu-ball** [comment]
Great to hear it's on the backlog, and given all the great stuff in the pipeline it's completely understandable that this one isn't a high priority!

**Comment by choigilhoon** [comment, marked as answer]
The AccessControl (SignedPolicy, AdmissionWebhooks) feature for thumbnail authentication has been updated in version v0.15.7.
For more information, please refer to this link (https://airensoft.gitbook.io/ovenmediaengine/access-control/admission-webhooks).

**Reply by stu-ball** [reply]
We've been using it for a while now and it's working exactly as we hoped it would. Thank you!!!
